torch/nn/functional.py

```diff
@@ -2834,7 +2834,7 @@ def pairwise_distance(x1, x2, p=2., eps=1e-6, keepdim=False):
 """)
 
 
-one_hot = _add_docstr(torch._C._nn.one_hot, r"""
+one_hot = _add_docstr(torch._C._nn.functional.one_hot, r"""
```

Comment:
This is an incorrect change. I think the right change would be to move `torch.nn.utils.one_hot` from Utilities section to the Functional section and under Sparse Functions in this [file](https://github.com/pytorch/pytorch/blob/master/docs/source/nn.rst). This should allow `one_hot` to be discoverable in the docs.

Reply:
Something like this:
```diff
diff --git a/docs/source/nn.rst b/docs/source/nn.rst
index f552ebb41..09fb41956 100644
--- a/docs/source/nn.rst
+++ b/docs/source/nn.rst
@@ -794,11 +794,6 @@ Utilities
 
 .. autofunction:: torch.nn.utils.remove_spectral_norm
 
-:hidden:`one_hot`
-~~~~~~~~~~~~~~~~~
-
-.. autofunction:: torch.nn.utils.one_hot
-
 
 .. currentmodule:: torch.nn.utils.rnn
 
@@ -1171,6 +1166,11 @@ Sparse functions
 
 .. autofunction:: embedding_bag
 
+:hidden:`one_hot`
+~~~~~~~~~~~~~~~~~
+
+.. autofunction:: one_hot
+
 Distance functions
 ----------------------------------
```